toolkit/scss/_search-result.scss

```diff
@@ -4,24 +4,34 @@
 @import "_shims.scss";
 
 .search-result {
-  margin: 0;
-  padding: 0 0 $gutter 0;
+
+  padding: 0 0 $gutter/2;
+  margin: 0 0 $gutter/2;
+
+  border-bottom: 1px solid $border-colour;
+
 }
 
 .search-result-title {
-  @include heading-24;
+
+  @include core-19;
```

Comment:
Could just use the `bold-19` mixin here: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss#L119